kolibri/plugins/management/assets/src/views/manage-content-page/channels-grid.vue

```diff
@@ -105,6 +112,11 @@
       this.currentTime = now();
       this.addChannelFileSummaries(map(this.channelList, 'id'));
     },
+    watch: {
+      channelList(val, newVal) {
```

Comment:
This is being called more often than I expected. Every poll there is a mutation to update the channel list. Even if there are no new channels, it seems that it's still signaling that channelList has 'changed'.